### ISSUE TITLE
change color of png for dark theme compatibility

### ISSUE THREAD

**Issue by loup-F** [zapthedingbat/drawio-obsidian#17]
Your work on this module is incredible and it would be extremely useful to be able to incorporate diagrams created on the fly into notes via embeds, unfortunately the resulting image seems to default to black lines and shapes, which makes it pretty hard to read on a dark mode themed obsidian. 

Would it be possible to add an option to make it so it is possible to use white or another color for embeds ? it would make this module perfect.

thank you for your work.

**Comment by zapthedingbat** [comment]
Yes, I've run into this same problem myself. It need a bit of investigation to work out the right way to handle it. It gets a bit confusing because the drawio editor has a dark theme too that seems to change the colors of the diagram.
